helpers.js

```diff
@@ -0,0 +1,16 @@
+function checkNumbers(...variables) {
```

Comment:
Naming should be more informative. Which numbers does this function check? How does it do it? 

Why are arguments called "variables"?